### PR TITLE
Contrib/runt9 illuminate integration

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

File: Playbooks/playbook-illuminate_Integration_Demonstration.yml [hunk 1]
```diff
@@ -0,0 +1,470 @@
+id: Illuminate Integration Demonstration
+version: -1
+name: Illuminate Integration Demonstration
+description: A quick demonstration of the various illuminate enrichment commands.
+fromversion: 1.8.7
```
Comment (yaakovi) [inline Playbooks/playbook-illuminate_Integration_Demonstration.yml]: random number?

File: Integrations/illuminate/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1 @@
+## [Unreleased]
```
Comment (yaakovi) [inline Integrations/illuminate/CHANGELOG.md]: remove this file

File: Playbooks/playbook-illuminate_Integration_Demonstration.yml (hunk 1 above)
Comment (ronykoz) [inline Playbooks/playbook-illuminate_Integration_Demonstration.yml]:
```suggestion

```

File: Integrations/illuminate/illuminate.yml [hunk 3]
```diff
@@ -0,0 +1,606 @@
...
+  Demisto indicators with data provided by the illuminate REST API, such as actor
+  and malware information, activity and reported dates, evidence and hit counts, and
+  more.
+configuration:
+- display: illuminate API Credentials (username/password)
+  name: credentials
+  defaultvalue: ''
+  type: 9
+  required: true
+- display: Domain of illuminate server to use
```
Comment (ronykoz) [inline Integrations/illuminate/illuminate.yml]:
```suggestion
- display: Domain of illuminate server to use (format: server.that_you_use.com - No http/https)
```

**Pull request merged**